aim/sdk/objects/figure.py

```diff
@@ -49,6 +53,8 @@ def _from_matplotlib_figure(self, obj):
             raise ModuleNotFoundError('Plotly is required to track matplotlib figure.')
 
         try:
+            logger.warning('Tracking a matplotlib object using "Figure" might not behave as expected.'
```

Comment:
What about just restricting tracking matplotlib figures with aim.Figure? If the majority of figures are not supported by plotly converter, it might make sense. @tmynn